tnc/daemon.py

```diff
@@ -369,7 +373,7 @@ def worker(self):
                         rigctld_ip=rigctld_ip,
                         rigctld_port=rigctld_port,
                     )
-
+                    hamlib.set_tx_delay(static.TX_DELAY)
```

Comment:
Wrong place. daemon just forwards data to main.py. This might cause the failing ctests